internal/mongodoc/doc.go

```diff
@@ -230,3 +242,35 @@ type Migration struct {
 	// Executed holds the migration names for migrations already executed.
 	Executed []string
 }
+
+// IntBool is a bool that will be represented internally in the database as 1 for
+// true and -1 for false.
+type IntBool bool
+
+func (b IntBool) GetBSON() (interface{}, error) {
+	if b {
+		return 1, nil
+	}
+	return -1, nil
+}
+
+func (b *IntBool) SetBSON(raw bson.Raw) error {
+	var x int
+	if err := raw.Unmarshal(&x); err != nil {
+		return errgo.Notef(err, "cannot unmarshal value")
+	}
+	switch x {
+	case 1:
+		*b = True
+	case -1:
+		*b = False
+	default:
+		return errgo.Newf("invalid value %q", x)
+	}
+	return nil
+}
+
+var (
```

Comment:
I'm not sure I'd bother with these tbh.
Just mongodoc.IntBool(true) would seem OK. Most of the places
you've used mongodoc.True and mongodoc.False can just
be "true" and "false" constants.

If you really think they do pull their weight then at least they should
be constants and not variables.